src/azul/queues.py

```diff
@@ -220,9 +220,11 @@ def wait_to_stabilize(self) -> int:
         Wait for the work queues to reach a steady state.
         """
         sleep_time = 10
+        timeout_limit = max(config.contribution_lambda_timeout(retry=True),
```

Comment:
Every timeout is a limit so why not just

```suggestion
        timeout = max(config.contribution_lambda_timeout(retry=True),
```